YoutubeCloneApp/Views/HomeView/HomeVideoCell.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  HomeVideoCell.swift
+//  YoutubeCloneApp
+//
+//  Created by 김지훈 on 2023/09/06.
+//
+
+import UIKit
+
+class HomeVideoCell: UICollectionViewCell {
+    static let identifier = "HomeVideoCell"
+    
+    //cell 스타일 지정?
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        cellPrint()
+    }
+    required init?(coder: NSCoder) {
+        fatalError("Not implemented required init?(coder: NSCoder)")
+    }
+    
+    lazy var thumbnailImage: UIImageView = {
```

Comment:
```suggestion
    private lazy var thumbnailImage: UIImageView = {
```

웬만하면 내부 view를 외부에 노출시키지 않는게 좋아요.

Reply:
확인했습니다.
@Publishable private(set) var videos: [YoutubeVideo] = [] 로 cell에 표시한 뒤 변경 예정입니다.

Reply:
youtubeService 연결 후 수정했습니다.